apps/examples/testcase/le_tc/drivers/tc_internal.h

```diff
@@ -0,0 +1,40 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/// @file tc_internal.h
+
+/// @brief Header file for Drivers TestCase Example
+#ifndef __EXAMPLES_TESTCASE_DRIVERS_TC_INTERNAL_H
+#define __EXAMPLES_TESTCASE_DRIVERS_TC_INTERNAL_H
+
+#include "tc_common.h"
+
+extern int total_pass;
+extern int total_fail;
+
+/**********************************************************
+* TC Function Declarations
+**********************************************************/
+void dev_null_main(void);
```

Comment:
I think dev_null and dev_zero should be in #ifdef~#endif condition

Reply:
Done